src/js/components/Icon/Icon.tsx

```diff
@@ -71,6 +72,15 @@ const Icon: React.FC<Props> = ({
       onMouseEnter={handleMouseEnter}
       onMouseLeave={handleMouseLeave}
     >
+      <defs>
+        <radialGradient id="instagram-gradient" r="150%" cx="30%" cy="107%">
+          <stop stopColor="#fdf497" offset="0" />
+          <stop stopColor="#fdf497" offset="0.05" />
+          <stop stopColor="#fd5949" offset="0.45" />
+          <stop stopColor="#d6249f" offset="0.6" />
+          <stop stopColor="#285AEB" offset="0.9" />
+        </radialGradient>
+      </defs>
```

Comment:
This defines a gradient with an Id that you can use with `fill: url(#id)`

Reply:
Nice solution!